lib/src/recovery_protocol/angular_analyzer_protocol.dart

```diff
@@ -218,7 +231,22 @@ class NgAnalyzerRecoveryProtocol extends RecoveryProtocol {
   @override
   RecoverySolution scanInterpolation(
       NgSimpleToken current, NgTokenReversibleReader reader) {
-    return new RecoverySolution.skip();
+    NgScannerState returnState;
+    NgToken returnToken;
+
+    if (current.type == NgSimpleTokenType.EOF ||
+        current.type == NgSimpleTokenType.commentBegin ||
+        current.type == NgSimpleTokenType.openTagStart ||
```

Comment:
Barriers for mustache error recovery is going to be fixed in a future PR (already done, just haven't submitted yet) to stop at mustache open, mustache close, EOF, and newline if the previous three cannot be found. It should be fine once that PR is submitted.